spark/benchmark/iceberg-source-orc-data-read-benchmark-result-after-vectorization.txt

```diff
@@ -0,0 +1,69 @@
+# JMH version: 1.21
```

Comment:
Can you remove the benchmark results? I'd rather not commit them to the repo.

Reply:
Yep, that was not intentional.